resolve-workgroup/decisions/AYD-RE-###-Decision-example.md

```diff
@@ -0,0 +1,47 @@
+# AYD-RE-### Decision Example
+
+- **Status:** Decided | Proposed | Rejected
+- **Impact:**  Low | Medium | High | Critical
+- **Contributors:** YNPUT Team <team@ynput.io>, Someone Else <someone.else@somewhere.to>
+- **Approved by:** John Doe <resposible@person.foo>
+
+## Background
+
+This is example decision record to be used by AYON workgroups as a template. In **Background** section should
+be a description of why the decision has to be made and any description that is important for it.
```

Comment:
I think the workgroup format lacks a `# Goals` section that describes what, in essence, it is that needs to be solved to consider the decision correct. Or maybe the current used "background" for many of the current 'decisions' lack these.

For example, a requirement might be: 

- API-access, 
- Able to re-run easily (even outside of publish)
- Automated tests (must allow for testing automation to secure future safety of the feature)

Reply:
I agree that **# Goals** is missing but I am assuming **## Action items** was meant to be alternative to it. @antirotor is that right?

Reply:
So the Goals were added since those I am assuming from description are not related to Action Items. Thanks @BigRoy for suggestion!